packages/flutter/lib/src/rendering/image.dart

```diff
@@ -161,8 +175,9 @@ class RenderImage extends RenderBox {
     if (constraints.isTight || _image == null)
       return constraints.smallest;
 
-    double width = _image.width.toDouble();
-    double height = _image.height.toDouble();
+    double scale = _scale ?? 1.0;
```

Comment:
I was going to allow null for consistency with width and height. I don't feel terribly strongly about it though.